pcb/elec/src/components/headers_rpi.ato

```diff
@@ -0,0 +1,32 @@
+"""
+Raspberry Pi headers.
+"""
+
+from "generics/interfaces.ato" import I2C
+
+from "rpi-header/elec/src/rpi.ato" import ConnectorRaspberryPi
+
+
+component HeaderGPIORaspberryPiTH from ConnectorRaspberryPi:
+    """
+    Raspberry Pi GPIO header.
+
+    Add pins not implemented in https://packages.atopile.io/package/rpi-header.
```

Comment:
happy to modify the package if you want to make a pull request!

Reply:
Done with ragnarlonn/rpi#5 and ragnarlonn/rpi#4